Code/Framework/AzFramework/AzFramework/Entity/EntityDebugDisplayBus.h

```diff
@@ -84,7 +89,13 @@ namespace AzFramework
         virtual void DrawWireHemisphere(const AZ::Vector3& pos, const AZ::Vector3& axis, float radius) { (void)pos; (void)axis; (void)radius; }
         virtual void DrawWireDisk(const AZ::Vector3& pos, const AZ::Vector3& dir, float radius) { (void)pos; (void)dir; (void)radius; }
         virtual void DrawBall(const AZ::Vector3& pos, float radius, bool drawShaded = true) { (void)pos; (void)radius; (void)drawShaded; }
-        virtual void DrawDisk(const AZ::Vector3& pos, const AZ::Vector3& dir, float radius) { (void)pos; (void)dir; (void)radius; }
+        virtual void DrawDisk(const AZ::Vector3& pos, const AZ::Vector3& dir, float radius, bool drawShaded = true)
+        {
+            (void)pos;
+            (void)dir;
+            (void)radius;
+            (void)drawShaded;
+        }
```

Comment:
And these.

And add the default true value for drawShaded that matches the previous behavior here as well.